packages/ndla-ui/src/Figure/component.figure.scss

```diff
@@ -200,9 +197,14 @@
     fill: $brand-color;
     color: $brand-color;
   }
-  @include mq($from: mobile) {
+}
+@media only screen and (max-width: 600px) {
```

Comment:
Fungerte ikke `@include mq($until: tablet)`?

Reply:
Legg også space over kodeblokken

Reply:
Ahh jo, det skal fungere. Brainfarta litt på syntax her. Setter den heller til `@include mq($until: desktop)` for å addressere problemet hvor det ikke skjer noe på bred mobilvisning.